db/migrate/20210322132613_create_reminder_experiments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateReminderExperiments < ActiveRecord::Migration[5.2]
+  def change
+    create_table :reminder_experiments do |t|
+      t.boolean :active, null: false
```

Comment:
In addition to `active`, I may also add a "type" field because there are two types of experiments. One for "active" patients and one for "inactive" patients and they behave differently. I may need to come up with better names for those groups or for this boolean, because it would be confusing to have an "active" type and an "active" bool. We need this boolean to indicate which experiment to look up when an appointment is created.

Reply:
We could also use a `state` enum with `["active", "inactive"]` allowed states instead of an `active` boolean. Will give us the ability to easily add more states if that becomes necessary.

Reply:
Not a bad idea.